src/assets/styles/baseStyles.scss

```diff
@@ -13,8 +13,14 @@ html{
   scroll-behavior: smooth;
 }
 
+
 body{
   font-family: 'Helvetica', sans-serif;
   font-size: 1.6rem;
-  margin: 0;
+  background-image: url('../images/background-light.png');
+  background-size: cover; 
+  background-position: center; 
+  background-repeat: repeat-y;
+  height: 300vh;
```

Comment:
Czemu to ma 300vh? nie powinno mieć sztywnej wysokości tylko automatycznie dopasować się do wielkosci ekranu i ew rozszerzyć if needed

Reply:
Jak ustawiałem na auto wysokość, to obraz się po prostu rozciągał przy większej ilości rzeczy na stronie. Dopiero jak ustawiłem stałą wysokość to się zaczął powielać

Reply:
@TheFenix2000

Reply:
sprawdziłem i wartość 100vh nie jest sztywna, przy dodawaniu kolejnych komponentów itemTile, zwiększa się view height, więc działanie jest prawidlowo, tylko lepiej zmienic z 300vh na 100vh żeby nie było mylące